RecoPixelVertexing/PixelVertexFinding/src/PVClusterComparer.cc

```diff
@@ -76,13 +76,12 @@ double PVClusterComparer::pTSquaredSum(const reco::Vertex &v) {
 }
 
 void PVClusterComparer::setChisquareQuantile() {
-  std::vector<double> maxChi2(20, 0.);
+  maxChi2_.clear();
+  maxChi2_.resize(20, 0.0);
   if (track_prob_min_ >= 0. && track_prob_min_ <= 1.)
-    for (size_t ndof = 0; ndof < maxChi2_.size(); ++ndof)
+    for (size_t ndof = 0; ndof < 20; ++ndof)
```

Comment:
Since you removed `maxChi2`, I think it is better to use directly
`     for (size_t ndof = 0; ndof < maxChi2_.size(); ++ndof) `